content/blog/why-is-redfish®-different-from-other-rest-apis-part-1.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Why is Redfish® different from other REST APIs - Part 1
+date: 2024-02-19T16:20:51.657Z
+author: François Donzé
+authorimage: /img/fdz-photoprofile.png
+thumbnailimage: ""
+disable: false
+tags:
+  - Redfish
+  - ilo-restful-api
+---
+## Redfish® brief recap
+
+Redfish® is a low level management RESTful API standardized by the Distributed Management Task Force (<a href="https://dmtf.org" target="_blank">DMTF</a>) consortium. DMTF publishes several standards related to server management, like the Common Information Model (CIM) implemented as OpenPegasus on Linux and Windows Management Instrumentation (WMI) on Microsoft Windows.
+
+Redfish® is born in August 2015. As of today, more than <a href="https://dmtf.org/about/list" target="_blank">sixty members</a> contribute to its development: computer makers, storage and network device manufacturers and software developers. A funny story says that "Redfish®" comes from the name of a restaurant near the Compaq drive in Houston, Texas, where initiators of the project were meeting before DMTF took over the entire project.
```

Comment:
If you keep this statement as is, you still need to change "is" to "was" (past tense). However, I think you might actually want to reword it to something like:
"Redfish made its first appearance in August 2015." or "Redfish was first introduced in August 2015."

Change "A funny story says that" to "A funny story relates how "Redfish" comes from..."